usaspending_api/accounts/models.py

```diff
@@ -165,7 +165,20 @@ class AppropriationAccountBalances(DataSourceTrackedModel):
     certified_date = models.DateField(blank=True, null=True)
     create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     update_date = models.DateTimeField(auto_now=True, null=True)
+    final_of_fy = models.BooleanField(blank=False, null=False, default=False, db_index=True)
 
     class Meta:
         managed = True
         db_table = 'appropriation_account_balances'
+
+    @classmethod
+    def populate_final_of_fy(cls):
+        sql = """   UPDATE appropriation_account_balances
```

Comment:
We're already pretty tied to Postgres as it is, so a little more piling on is probably fine.